### PR TITLE
Adding colors to the admin panel

### PR DESCRIPTION
**Pull request by alejandroGonGon**
Adding the main colors of the web app and modifying the styles of the admin panel.

Demo :

https://www.loom.com/share/81c83780d83f451294ca74e963a8217f

### CODE REVIEW

File: app/Orchid/Screens/PlatformScreen.php [hunk 1]
```diff
@@ -41,18 +39,11 @@ public function query(): array
      */
     public function commandBar(): array
     {
+        $url = "http://ec2-3-145-124-177.us-east-2.compute.amazonaws.com/";
```
Comment (nicocadq) [inline app/Orchid/Screens/PlatformScreen.php:42]: Could we move this to an env? In order to change it in each environment

File: resources/views/brand/header.blade.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <link
+        href="/favicon.ico"
+        id="favicon"
+        rel="icon"
+    >
+@endpush
+
+<p class="h2 n-m font-thin v-center">
+    <span class="m-l d-none d-sm-block">
+        Panel Administrativo
```
Comment (nicocadq) [inline resources/views/brand/header.blade.php:11]:
```suggestion
Admin Panel
```

File: resources/views/brand/header.blade.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        <small class="v-top opacity">hoydonamos</small>
```
Comment (nicocadq) [inline resources/views/brand/header.blade.php:12]: Why just `hoydonamos`?

File: app/Orchid/Screens/PlatformScreen.php (hunk 1 above)
Comment (alejandroGonGon) [reply app/Orchid/Screens/PlatformScreen.php:42]: UH SI, como no se me ocurrio!

File: resources/views/brand/header.blade.php (hunk 3 above)
Comment (alejandroGonGon) [reply resources/views/brand/header.blade.php:12]: Looks simple and stetic but this is relative, would you like to change it ?

**Review by nicocadq — APPROVED** [review]

**Pull request merged**